public/locales/fr/translation.json

```diff
@@ -509,8 +510,8 @@
         "properties.keywords": "Mots-clés",
         "properties.landscape-entities": "Entités paysagères",
         "properties.related-elements": "Documents associés",
-        "geometry.coordinates[0]": "Longitude",
-        "geometry.coordinates[1]": "Latitude"
+        "point.coordinates[0]": "Longitude",
+        "point.coordinates[1]": "Latitude"
```

Comment:
Maybe it's preferable to be more verbose such as :
"point.coordinates.lng": "Longitude",
"point.coordinates.lat": "Latitude"

Reply:
Back-end API provide only an array of coordinates and we are using it "as is".